src/sentry/integrations/base.py

```diff
@@ -463,3 +468,72 @@ def notify_remove_external_team(self, external_team: ExternalActor, team: Team)
 
     def remove_notification_settings(self, actor_id: int, provider: str) -> None:
         pass
+
+
+def is_response_success(resp) -> bool:
+    if resp.status_code and resp.status_code < 300:
+        return True
+    return False
+
+
+def is_response_error(resp) -> bool:
```

Comment:
same comment here

Reply:
Also, would this function be the opposite of the function above? Or is there a scenario where it's not an error or success?

Reply:
i don't know, i'm copying what the previous person did. it seems like they don't count 429s or 500s as meaning the integration is broken https://www.notion.so/sentry/Integration-Auto-Disable-Notify-5266cce6e5834b1d8120cfde93050fb8

Reply:
It's been a while but I think we did this to avoid disabling the integration in case there are temporary issues